bench/perf_run_py/perf_run.py

```diff
@@ -75,18 +79,16 @@ def get_git_info(git_working_tree_dir):
     return git_info
 
 
-def construct_wtperf_command_line(wtperf: str, env: str, test: str, home: str, arguments: List[str]):
+def construct_command_line(exec_path: str, test: str, home: str, arguments: List[str]):
     command_line = []
-    if env is not None:
-        command_line.append(env)
-    command_line.append(wtperf)
+    command_line.append(exec_path)
     if test is not None:
-        command_line.append('-O')
+        command_line.append('-O') if is_wtperf_test(exec_path) else None
```

Comment:
Having an if/else at this point is likely to lead to complex/confusing code down the track. The choice of using '-O' or nothing (or '-h' or '-home' below) is based only on the type of test (and the test executable).
One idea would be putting the if/then at a much higher level in the code (and based on a command line parameter rather than matching a string) and passing the required strings through appropriately from the higher level (perhaps together in an object) - that will avoid having these if/else statements deep in the code.